src/pages/resultsView/mutation/Mutations.tsx

```diff
@@ -6,6 +6,7 @@ import ResultsViewMutationMapper from './ResultsViewMutationMapper';
 import { convertToMutationMapperProps } from 'shared/components/mutationMapper/MutationMapperConfig';
 import MutationMapperUserSelectionStore from 'shared/components/mutationMapper/MutationMapperUserSelectionStore';
 import { observable } from 'mobx';
+import { PatientViewPageStore } from '../../../pages/patientView/clinicalInformation/PatientViewPageStore';
```

Comment:
again, why do we need `PatientViewStore` in a component defined for results view page?

Reply:
doesnt appear to even be used so it'll be removed